fairroot/base/sink/FairSink.h

```diff
@@ -40,7 +40,7 @@ class FairSink
     virtual ~FairSink();
 
     virtual Bool_t InitSink() = 0;
-    virtual void Close() = 0;
+    [[deprecated("Please close your resources in derived class dtor")]] virtual void Close();
```

Comment:
1. Well, sadly enough the deprecation message is only shown, when calling the function. So we should have something for people calling it.
2. If you don't say `= 0;`, you promise to implement it somewhere. Let's just implement it inline!

```suggestion
    [[deprecated("Close will go away, the dtor closes resources")]] virtual void Close() {}
```

Reply:
Fixed